File: app/src/main/java/top/itning/yunshuclassschedule/util/EventReceiver.java
package top.itning.yunshuclassschedule.util;

/**
 * 事件接收
 *
 * @author itning
 */
public interface EventReceiver {
    /**
     * 触发事件
     *
     * @return 已消费返回真
     */
    boolean eventTrigger();
}

File: app/src/main/java/top/itning/yunshuclassschedule/util/EventReceiver.kt
package top.itning.yunshuclassschedule.util

/**
 * 事件接收
 *
 * @author itning
 */
interface EventReceiver {
    /**
     * 触发事件
     *
     * @return 已消费返回真
     */
    fun eventTrigger(): Boolean
}
